src/func.cc

```diff
@@ -616,7 +616,7 @@ void CallFunc(const vector<Value*>& args, Evaluator* ev, string* s) {
   for (size_t i = 1; i < args.size(); i++) {
     unique_ptr<SimpleVar> s(
         new SimpleVar(args[i]->Eval(ev), VarOrigin::AUTOMATIC, nullptr, Loc()));
-    av.push_back(move(s));
+    av.push_back(std::move(s));
```

Comment:
probably a better way of addressing this is to 
```
av.emplace_back(std::make_unique<SimpleVar>(
                               args[i]->Eval(ev), VarOrigin::AUTOMATIC, nullptr, Loc())));
```

Reply:
Done.